Makefile

```diff
@@ -591,6 +591,7 @@ ifdef ASSERT_STATUS_CHECKED
 		db_iter_stress_test \
 		db_log_iter_test \
 		db_bloom_filter_test \
+		db_blob_corruption_test \
```

Comment:
Nit: could you put this in between `db_blob_basic_test` and `db_blob_index_test` to keep the alphabetical order?